frontend/public/components/dashboard/project-dashboard/utilization-card.tsx

```diff
@@ -136,6 +200,24 @@ export const UtilizationCard = withDashboardResources(
             error={memoryError}
             TopConsumerPopover={memPopover}
           />
+          <UtilizationItem
+            title="Filesystem"
+            data={filesystemStats}
+            isLoading={!projectName || !filesystemUtilization}
+            humanizeValue={humanizeBinaryBytes}
+            query={queries[ProjectQueries.FILESYSTEM_USAGE]}
+            error={filesystemError}
+            TopConsumerPopover={filesystemPopover}
+          />
+          <UtilizationItem
+            title="Network"
```

Comment:
```suggestion
            title="Network Transfer in/out"
```

It looks like this number is the sum of both inbound and outbound network traffic, so this label might be slightly more clear. Do you agree? Something like this seems fine until we can have two separate lines like our mockups:

![image](https://user-images.githubusercontent.com/9122899/68696523-21719780-054b-11ea-8241-8354148106cb.png)